src/lens/Quoter.sol

```diff
@@ -55,7 +55,7 @@ contract Quoter is IQuoter, SafeCallback {
     constructor(IPoolManager _poolManager) SafeCallback(_poolManager) {}
 
     /// @inheritdoc IQuoter
-    function quoteExactInputSingle(QuoteExactSingleParams memory params)
+    function quoteExactInputSingle(QuoteExactSingleParams calldata params)
```

Comment:
in the OZ issue, i think we wanted memory on both implementation and interface. I think there were incidents in IQuoter that were using calldata.

Because we are redoing Quoter entirely, i dont think we need to change it back